src/PrestaShopBundle/Resources/config/services/core/form/command_builder.yml

```diff
@@ -66,8 +66,6 @@ services:
     arguments:
       - !tagged core.combination_command_builder
 
-  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\MultiShopCombinationCommandsBuilderInterface: '@PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\CombinationCommandsBuilder'
```

Comment:
If you keep this it will define the default service for this interface thus it would simplify many other service definitions with autowiring as ou don't need to explicitly set them
But it's not blocking

Reply:
We require same interface in so many places and it is designed to contain many usages, so I think its better we don't provide a default service for this interface, but always be explicit. (to avoid mistakes auto injecting wrong one)

Reply:
Ok I understand your worry